src/feature/extraction.cc

```diff
@@ -64,6 +91,21 @@ SiftFeatureExtractor::SiftFeatureExtractor(
       image_reader_(reader_options_, &database_) {
   CHECK(reader_options_.Check());
   CHECK(sift_options_.Check());
+  
+  std::shared_ptr<Bitmap> camera_mask;
+  if (!reader_options_.camera_mask_path.empty()) {
+    camera_mask = std::shared_ptr<Bitmap>(new Bitmap(), [](Bitmap* b){
+      b->Deallocate();
```

Comment:
This shouldn't be necessary, because the destructor of Bitmap deallocates the data already.